examples/Responsive.js

```diff
@@ -10,6 +21,7 @@ export default class Responsive extends Component {
       slidesToShow: 4,
       slidesToScroll: 4,
       initialSlide: 0,
+      unslick: UnSlick,
```

Comment:
@akiran I think its pretty necessary.  `react-slick` decides when to unslick at runtime based on responsive widths and its nice to not duplicate that logic both outside/inside the slider.